terraform-docker/apply/main.yaml

```diff
@@ -79,7 +141,14 @@ stages:
                           sudo chown -R $(id -u):$(id -g) ${PWD}/${DIR}
                           sudo chown -R $(id -u):$(id -g) ${AZURE_CONFIG_DIR}
                           sudo chown -R $(id -u):$(id -g) ${PWD}/global.tfvars
+
+                          if [[ AWS_REGION != "" ]]
+                          then
```

Comment:
I'm unsure why we do this chown, my guess is that it's for a security reason to not keep the files with so much access after the pipeline is run. 
So I copied the current solution but with a if  so we still can be run a Azure only env.

Reply:
@NissesSenap issue with different users outside and inside of the container - if we don't do this, there's going to be an issue with the agent cleaning up the temporary folder when the job is completed (won't have access) and subsequent runs on the same agent will fail due to this.